src/grpc.ts

```diff
@@ -63,6 +63,10 @@ export default class GrpcClient {
         })
     }
 
+    get refreshToken() {
+        return this.refresh_token
```

Comment:
Instead of using an accessor, let's just use a standard method
```js
getRefreshToken() {
  return this.refresh_token
}
```

Reply:
We don't tend to (or even at all) use accessors in this library, so I'd rather we stick to the same style. Ideally, we should add some documentation on contribution/styleguide.

Reply:
Got it, thank you :)